actions/docs-verifier/src/MarkdownLinksVerifier/LinkClassifier/Classifier.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace MarkdownLinksVerifier.LinkClassifier
+{
+    internal static class Classifier
+    {
+        public static LinkClassification Classify(string link)
```

Comment:
```suggestion
        internal static LinkClassification Classify(string link)
```

If the class is `internal`, why would the method be `public`?